runeq/v2sdk/stream.py

```diff
@@ -6,42 +6,55 @@
 from . import client
 
 
-def get_stream(stream_id, **params):
+def get_stream(stream_id: str, **params):
```

Comment:
It seems to me it would be preferable to have separate parameters (with type annotations) so the SDK supports static type analysis (and is a bit more self-documenting). This should also help IDEs and such spot errors early (which would be help the UX I imagine, since this library is intended for non-developers as well). The code below can combine those into a dict for the purposes of making the GET request. This method could also have a return type.

This comment applies to `get_availability` below as well.